components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -473,7 +473,9 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     [self.itemViews addObject:itemView];
     [self.containerView addSubview:itemView];
   }
+  self.selectedItem = nil;
```

Comment:
I believe this would be a change in behavior for clients. Does it make sense to check if the previous `selectedItem` is in the current `items` array and only deselect it if it's absent?

Reply:
Without this line you get unexpected behavior, where you can't select the item that was previously selected and it doesn't show itself as selected. The default behavior for UITabBar is to reset to nil so that's why we have this.

Reply:
Ack.